solidity/random-beacon/test/RandomBeacon.test.ts

```diff
@@ -137,8 +139,8 @@ describe("RandomBeacon", () => {
       })
 
       it("should update the DKG submission eligibility delay", async () => {
```

Comment:
"should update the DKG submission eligibility delay" => "should update the DKG result submission eligibility delay"